GithubJobsEnterpriseProject/ClientApp/src/components/Register.js

```diff
@@ -1,37 +1,93 @@
-import React, {useState,useEffect} from 'react'
+import React, { useState, useEffect } from 'react'
+import axios from 'axios';
+import Login from './Login';
 
 export default function Register() {
-    const [user, getUser] = useState([]);
+
+    const [username, setUsername] = useState([]);
+    const [password, setPassword] = useState([]);
+    const [passwordRe, setPasswordRe] = useState([]);
+    const [email, setEmail] = useState([]);
+    const [isValidUsername, setIsValidUsername] = useState(false);
+    const [isValidEmail, setIsValidEmail] = useState(false);
+    const [isCredentailsValid, setIsCredentailsValid] = useState(null);
+
+    const handleChange = (e) => {
+        if (e.target.name == "Username") {
+            if (e.target.value !== null && e.target.value.length < 6) {
+                setUsername(e.target.value);
+                setIsValidUsername(false);
+            } else {
+                setUsername(e.target.value);
+                setIsValidUsername(true);
+            }
+        }
+        else if (e.target.name == "Email") {
+            if (e.target.value !== null && e.target.value.length < 12) {
+                setUsername(e.target.value);
+                setIsValidEmail(false);
+            } else {
+                setEmail(e.target.value);
+                setIsValidEmail(true);
+            }
+        }
+        else if (e.target.name == "Password") {
+            setPassword(e.target.value);
+        } else {
+            setPasswordRe(e.target.value);
+        }
+
+    };
 
     const handleSubmit = (event) => {
-        event.preventDefault()
-        
+        event.preventDefault();
+        axios.get("/register/username=" + username + "&email=" + email + "&password=" + password).then(data => { setIsCredentailsValid(data.data) });
```

Comment:
Use string template instead of concatenating strings.
```diff
-"/register/username=" + username + "&email=" + email + "&password=" + password
+`/register/username=${username}&email=${email}&password=${password}`
```